Hivemind/Assets/Scripts/Ui/UiController.cs

```diff
@@ -1,19 +1,24 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
 using Assets.Scripts;
 using TMPro;
 using UnityEngine;
 using UnityEngine.EventSystems;
 
-public class UiController : MonoBehaviour, IInitializePotentialDragHandler, IDragHandler, IEndDragHandler
+public class UiController : MonoBehaviour, IInitializePotentialDragHandler, IDragHandler, IEndDragHandler, IPointerClickHandler
 {
     private MindGroup currentOpenMindGroup;
 
     private Vector2 lastMousePosition; // Used in calculating screen drag of icons
 
     [SerializeField] private GameObject mindBuilderPanel;
 
+    [SerializeField] private GameObject mainCamera;
```

Comment:
Deze kan je vinden aangezien er maar 1 maincamera object per scene aanwezig is, heb je ook minder problemen met een nieuwe map aanmaken waar je dit handmatig moet invullen

Reply:
dit doe je namelijk ook met minimaps